harness/determined/cli/task.py

```diff
@@ -217,6 +250,35 @@ def logs(args: Namespace) -> None:
                     *common_log_options,
                 ],
             ),
+            Cmd(
+                "create",
+                create,
+                "create task",
+                [
+                    Arg("config_file", type=FileType("r"), help="task config file (.yaml)"),
+                    Arg(
+                        "context",
+                        type=Path,
+                        help="file or directory containing task context directory",
+                    ),
+                    Arg(
+                        "-i",
+                        "--include",
+                        action="append",
+                        default=[],
+                        type=Path,
+                        help="additional files to copy into the task container",
+                    ),
+                    Arg("--project_id", type=int, help="place this task inside this project"),
+                    Arg("--config", action="append", default=[], help="TODO HELP"),
```

Comment:
I think we should be able to reuse the `CONFIG_DESC` from https://github.com/determined-ai/determined/blob/main/harness/determined/cli/command.py#L19